sentry/0001-kiwicom-login.patch

```diff
@@ -1,23 +1,23 @@
---- templates/sentry/login.html 9.0.0
+--- templates/sentry/login.html 9.1.0
```

Comment:
you even updated the path comment here 😍

Reply:
❤️